v3/plugins/redhat/vscode-camelk/latest/meta.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+publisher: redhat
+name: vscode-camelk
+version: latest
+type: VS Code extension
+displayName: Tooling for Apache Camel K
+title: Tooling for Apache Camel K
+description: VS Code support for Apache Camel K functionality
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/camel-tooling/vscode-camelk
+category: Language
+firstPublicationDate: '2019-07-10'
+spec:
+  containers:
+  - image: "eclipse/che-remote-plugin-camelk-0.0.9:next"
```

Comment:
Please use quay.io and in general we should always include the registry here, not just the `org/image:tag`.